### PR TITLE
adding youtube channel link to side menu

### PR DESCRIPTION
**Pull request by SamFritz**
This pull request adds in the new archives unleashed youtube channel link to the side menu on the website and addresses [issue #53 ](https://github.com/archivesunleashed/archivesunleashed.org/issues/53). 

Please note, at this time I've been unable to locate solution that allows for added icon in header. Attempts were made by configuring config.toml file and searches for examples of adding icon to hugo material docs template.

### CODE REVIEW

**Comment by ianmilligan1** [comment]
Thanks for this, @SamFritz! 

I don't know if our YouTube channel is enough of a site of activity yet to have it on our left sidebar, which already has a number of items on it.

screen shot 2018-09-18 at 3 29 22 pm

I might suggest instead adding the link to the "Get Involved" and perhaps embedding the walkthrough video somewhere on the Archives Unleashed Cloud page?

**Comment by SamFritz** [comment]
Sounds good! Will add into subscribe page for sure and see about best way to integrate into AUK page. 

Thanks for the review

File: content/cloud/index.md [hunk 1]
```diff
@@ -29,7 +29,12 @@ In addition, AUK's [documentation](http://cloud.archivesunleashed.org/documentat
 
 ## A Guided Tour of AUK
 
-Let's take a tour of how AUK works. Once a user signs up to [cloud.archivesunleashed.org](http://cloud.archivesunleashed.org/), they enter their Archive-It credentials, which are salted and encrypted. Those credentials are then used to sync their Archive-It collections with AUK using Archive-It's [WASAPI](https://github.com/WASAPI-Community/data-transfer-apis) endpoint. This is done as a background job, and once it is complete, it emails the user to let them know that their Archive-It collections are synced and available for further analysis. 
+
+Let's take a tour of how AUK works. Below you will find a video tour of the Archives Unleashed Cloud, followed by a brief overview of what you can expect when working with AUK. 
+
```
Comment (ianmilligan1) [inline content/cloud/index.md]: Right now the iframe hugs this paragraph above it a bit awkwardly, i.e.

screen shot 2018-09-25 at 1 32 06 pm

The usual Markdown ways to force a break aren't working (i.e. two spaces) but by manually putting in two line breaks I think it looks a bit better. This is with `<br/><br/>` added at the end of the sentence above.

screen shot 2018-09-25 at 1 33 17 pm

File: content/get-involved/index.md [hunk 2]
```diff
@@ -12,12 +12,14 @@ There are many ways you can participate in the Archives Unleashed Project. You c
...
+| ![Share](/images/share.png)              | Love web archives? Share our website, project, events, and beyond with colleagues.                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ![Learn](/images/learn.png)             | Checkout our learning resources, including a new [YouTube Channel](https://www.youtube.com/channel/UC4Sq0Xi6UWhYK2VbmAzFhAw). The AUK [Learning Guides](https://cloud.archivesunleashed.org/derivatives) walk through how to use derivative files. If you'd like more hands on with AUT, examples scripts can be found in the [Archives Unleashed Toolkit](https://archivesunleashed.org/aut/) section of our website. |
```
Comment (ianmilligan1) [inline content/get-involved/index.md]: Let's spell out "AUK" as "Archives Unleashed Cloud" and AUT as "Archives Unleashed Toolkit." 

Maybe the second mention of "Archives Unleashed" (which is a hyperlink) could use become "... can be found in this section of our website ..."

File: content/cloud/index.md (hunk 1 above)
Comment (SamFritz) [reply content/cloud/index.md]: Thanks @ianmilligan1, I was playing around with this but couldn't figure out how to force the break!

**Pull request merged**